docs/manual-EN/server-administration/storage-balance.md

```diff
@@ -0,0 +1,219 @@
+| 192.168.8.210 | 34600 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34700 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34500 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+Got 3 rows (Time spent: 5886/6835 us)
+```
+
+**_Explanations on the returned results:_**
+
+- **IP** and **Port** is the present storage instance, the cluster started three storaged instances without any data. (192.168.8.210:34600, 192.168.8.210:34700, 192.168.8.210:34500)
+- **Status** shows the state of the present instance, currently there are two kind of states, i.e. online/offline. When a machine is out of service, metad will turn it to offline if no heart beat received for certain time threshold. The default threshold is 10 minutes and can be changed in parameter `expired_threshold_sec` when starting metad service.
+- **Leader count** shows RAFT leader number of the present process.
+- **Leader distribution** shows how the present leader is distributed in each graph space, no space is created for now.
+- **Partition distribution** is the partition number in different spaces.
+#### Step 1.2
+Create a graph space named **test** with 100 partition and 3 replicas.
+
+```
+(user@127.0.0.1) [(none)]> CREATE SPACE test(PARTITION_NUM=100, REPLICA_FACTOR=3)
+
+Execution succeeded (Time spent: 2770/3777 us)
+```
+
+Show partition distribution of the cluster using command `SHOW HOSTS` seconds later.
+
+```
+(user@127.0.0.1) [(none)]> SHOW HOSTS
+
+================================================================================================
+| Ip            | Port  | Status | Leader count | Leader distribution | Partition distribution |
+================================================================================================
+| 192.168.8.210 | 34600 | online | 0            | test: 0             | test: 100              |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34700 | online | 52           | test: 52            | test: 100              |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34500 | online | 48           | test: 48            | test: 100              |
+------------------------------------------------------------------------------------------------
+Got 3 rows (Time spent: 6029/7175 us)
+```
+
+### Step 2 Start new storage instances
+Start 5 new storaged instances to scale out, show the new status using command `SHOW HOSTS`.
```

Comment:
Now, add some new hosts (storaged instances) to the cluster.

Again, show the new status using command `SHOW HOSTS`. You can see there are already eight hosts in serving. But no partition is running on the new hosts.